.github/workflows/ci.yaml

```diff
@@ -15,15 +15,9 @@
 name: CI
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
   push:
     branches:
+      - "pull-request/[0-9]+"
```

Comment:
This change still doesn't make sense to me.  How would this constraint even be satisfied when the PR branches aren't named this way?

Reply:
Me too. @ArangoGutierrez do you know why this change was initially made?

Reply:
When the copy-pr bot is enabled it creates a branch named `pull-request/<PR_ID>` (for example https://github.com/NVIDIA/gpu-operator/tree/pull-request/1095 for this PR).

The bot was enabled in https://github.com/NVIDIA/gpu-operator/pull/1068.